packages/x-date-pickers/src/internals/components/PickersPopper.tsx

```diff
@@ -284,6 +285,7 @@ export const PickersPopper = (props: PickerPopperProps) => {
   };
 
   const ActionBar = components?.ActionBar ?? PickersActionBar;
+  const PopperPaperContent = components?.PopperPaper || React.Fragment;
```

Comment:
Should we rename it to be explicit on the fact it does not override the `Paper`?
```suggestion
  const PopperPaperContent = components?.PopperPaperContent || React.Fragment;
```

Reply:
Renamed the component to `PaperContent` and made sure it is propagated to the `Static` pickers as well. 😉